backend/src/main/resources/db/migration/V1__Initial_schema.sql

```diff
@@ -0,0 +1,9 @@
+-- Users table
+CREATE TABLE users (
+    id INT PRIMARY KEY NOT NULL,
```

Comment:
If you don’t expect more than ~2.1 billion users (max value of signed INT) this should be fine 😆 
But usually we use `UUID` (Based on LabTwin and Voyfai experience)